### DISCUSSION TITLE
[IntelliSense] Add icon for Tailwind CSS language mode

### DISCUSSION THREAD

**Discussion by danteissaias** [tailwindlabs/tailwindcss#10333] [Ideas]
<img width="588" alt="image" src="https://user-images.githubusercontent.com/105590557/212689333-c8dceb44-b718-4b9f-b63a-1bab9792eb1e.png">

**Comment by aenial51** [comment]
Hello
here you can get whole information 
[Tailwind CSS Tutorial #14 - Icons - YouTube](https://www.youtube.com/watch?v=aNmBiqK2uQ0)
visit this page .

[Nexus Iceland
](https://www.nexusiceland.org/)

**Comment by 52461** [comment]
Hello,
 Drop the SVG file or paste the SVG markup into the optimizer at SVGOMG.
Copy the markup provided by SVGOMG and paste into your project.
Remove any fill or stroke attributes so Tailwind can modify those with classes (not shown in the video).   [Home Bargains Portal](https://www.homebargainsportal.com/)
